common/src/main/scala/db/DatabaseConfig.scala

```diff
@@ -38,6 +39,11 @@ object DatabaseConfig {
     hikariConfig.setJdbcUrl(config.url)
     hikariConfig.setUsername(config.user)
     hikariConfig.setPassword(config.password)
+
+    val dsProperties: Properties = new Properties()
+    dsProperties.setProperty("socketTimeout", "30")
```

Comment:
Could this be achieved with `hikariConfig.addDataSourceProperties("socketTimeout", "30")` instead ? (note the `add` rather than `set`

Reply:
Ha, you're probably right!